sdk/proto/analyzer.proto

```diff
@@ -116,13 +116,15 @@ message AnalyzeDiagnostic {
     repeated string tags = 6;              // Keywords/terms to associate with a policy, e.g., "cost".
     EnforcementLevel enforcementLevel = 7; // Severity of the policy violation.
     string urn = 8;                        // URN of the resource that violates the policy.
+    string policyPackVersionTag = 9;       // Version Tag of the Policy Pack.
```

Comment:
Why is it necessary to add this? Why couldn't the existing `policyPackVersion` be used (which effectively was never really hooked up in the policy SDK)?